vms/platformvm/txs/camino_address_state_tx.go

```diff
@@ -71,6 +79,13 @@ func (tx *AddressStateTx) SyntacticVerify(ctx *snow.Context) error {
 		return nil
 	case tx.Address == ids.ShortEmpty:
 		return ErrEmptyAddress
+	case codec.GetUpgradeVersion(tx.UpgradeVersionID) > 0:
```

Comment:
would be nice to have syntactic-test test cases added in this PR

Reply:
done